DEVELOPING.md

```diff
@@ -53,12 +53,18 @@ If you are on a Mac, you will need brew, direnv, habitat, and Docker for Mac
 
 ### Studio requirements
 
-1. Ensure you have run `hab setup` in your environment that will be executing the studio
+Ensure you have run `hab setup` in the environment that will be executing the studio, or exported environment variables of `HAB_ORIGIN` and `HAB_AUTO_TOKEN` whose values correspond with the public Builder service. For example:
+
+    ```
+    export HAB_ORIGIN=<YOUR_PERSONAL_ORIGIN>
```

Comment:
The origin may need to be 'habitat' - the package builds within dev studio doesn't seem to work otherwise.

Reply:
I was able to `bb` everything with `HAB_ORIGIN` set as my personal, but setting it as `habitat` makes more sense, anyway. I'll update.